internal/backup_push_handler.go

```diff
@@ -110,15 +110,16 @@ func HandleBackupPush(uploader *Uploader, archiveDirectory string) {
 	bundle := NewBundle(archiveDirectory, crypter, previousBackupSentinelDto.BackupStartLSN, previousBackupSentinelDto.Files)
 
 	var meta ExtendedMetadataDto
-	meta.StartTime = time.Now()
+	meta.StartTime = utility.CeilTimeUpToMicroseconds(time.Now())
```

Comment:
What about a single function for:
utility.CeilTimeUpToMicroseconds(time.Now())
?